configuration/ldap/extra.py

```diff
@@ -0,0 +1,8 @@
+####
+## This file contains extra configuration options that can't be configured
+## directly through environment variables.
+####
+
+# import ldap
```

Comment:
Is this here by accident or intentionally?

Reply:
Are you referring to the whole file or the `import ldap`?

Reply:
The `import ldap` snippet.

Reply:
I don't think you've answered my question?